ios/Capacitor/Capacitor/Plugins/Camera.swift

```diff
@@ -111,7 +111,7 @@ public class CAPCameraPlugin : CAPPlugin, UIImagePickerControllerDelegate, UINav
     }))
 
     alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: { (action: UIAlertAction) in
-      alert.dismiss(animated: true, completion: nil)
+      self.imagePickerControllerDidCancel(self.imagePicker!)
```

Comment:
Better just call the error directly
```suggestion
      self.call?.error("User cancelled photos app")
```
`imagePickerControllerDidCancel` should only be called by the delegate, and also it dismiss the imagePicker, but at this point it has not been presented.